app/models/pipeline_run.rb

```diff
@@ -299,18 +299,34 @@ def db_load_ercc_counts
   end
 
   def db_load_amr_counts
-    amr_results = PipelineRun.download_file(s3_file_for("amr_counts"), local_amr_full_results_path)
+    amr_drug_mappings = {}
+    amr_drug_mappings["AGly"] = "aminoglycosides"
+    amr_drug_mappings["Bla"] = "beta-lactamases"
+    amr_drug_mappings["Fos"] = "fosfomycin"
+    amr_drug_mappings["Flq"] = "fluoroquinolones"
+    amr_drug_mappings["Gly"] = "glycopeptides"
+    amr_drug_mappings["MLS"] = "macrolide-lincosamide-streptogarmin"
+    amr_drug_mappings["Phe"] = "phenicols"
+    amr_drug_mappings["Rif"] = "rifampicin"
+    amr_drug_mappings["Sul"] = "sulfonamides"
+    amr_drug_mappings["Tet"] = "tetracyclines"
+    amr_drug_mappings["Tmt"] = "trimethoprim"
+    # amr_results = PipelineRun.download_file(s3_file_for("amr_counts"), local_amr_full_results_path)
+    amr_results = PipelineRun.download_file("s3://idseq-database/test/AMR/amr_processed_results.csv", local_amr_full_results_path)
     unless File.zero?(amr_results)
       amr_counts_array = []
       # First line of output file has header titles, e.g. "Sample/Gene/Allele..." that are extraneous
       # that we drop
       File.readlines(amr_results).drop(1).each do |amr_result|
         amr_result_fields = amr_result.split(",").drop(2)
+        drug_family_key = amr_result_fields[12]
+        drug_expansion = amr_drug_mappings[drug_family_key]
+        formatted_drug_family = "#{drug_family_key} (#{drug_expansion})"
```

Comment:
You should not store this information in the database like that.
(a) drug_family column should continue to contain just the 3-letter identifier. We'll probably want to query it for other applications later (perhaps with conditions like `where(drug_family: "AGly")`), so the cosmetic change has no place here
(b) You don't need to store the full name over and over in the database -- you can add the display name as part of the `show` action of the controller to the hash exposed as @amr_counts. You can put the function that does the data massaging into PipelineRunHelper (don't write it out inside samples_controller)